resources/web/docs_js/index.js

```diff
@@ -194,4 +194,11 @@ $(function() {
   } else {
     init_toc(LangStrings);
   }
+
+  // Setup hot module replacement for css if we're in dev mode.
```

Comment:
This is fairly hacky but I don't want to change the template when building a single book just to include this. I could be convinced though.